config/deploy.rb

```diff
@@ -40,3 +40,21 @@
 
 set :honeybadger_env, fetch(:stage)
 
+
+Rake::Task["honeybadger:deploy"].clear
+namespace :honeybadger do
+  desc 'Notify Honeybadger of the deployment.'
+  task :deploy => [:'deploy:set_current_revision'] do
+    puts "Temoporarily disabling honeybadger deploy notification, until we fix it."
+  end
+end
+
+namespace :deploy do
```

Comment:
Based on your demonstration that the other task needed a `clear`, can you just delete this second section, since it isn't changing anything?